cargo/src/main.rs

```diff
@@ -1,25 +1,18 @@
 {% unless std -%}
 #![no_std]
 #![no_main]
-{% endunless -%}
-use esp_idf_sys as _; // If using the `binstart` feature of `esp-idf-sys`, always keep this module imported
-{%- if std and hal != "No" %}
-use log::*;
-{% endif %}
 
-{%- unless std %}
+{% endunless -%}
+{% unless std -%}
 #[no_mangle]
-{%- endunless %}
+{% endunless -%}
```

Comment:
Cant this conditions be joined?

Reply:
Same here. Don't want to touch it too much. biggest hurdle is to get the whitespacing right so `cargo fmt` does not complain.